methodologies/ricochet-programming/README.md

```diff
@@ -0,0 +1,45 @@
+# Ricochet Programming
+
+by @pawelpluta
+
+The subtle, but very powerful technique, allowing you to boost up the development process and save your time for another round of Tekken 3.
+Ricochets are specialized tools for resolving small tasks and even user stories at almost zero cost. The smaller tasks are,
+the bigger benefit can be achieved. Ricochets are not very accurate, therefore they cannot be always 
+used, but when there are multiple targets, the probability of hitting increases.
+
+Let's understand the process of triggering ricochets:
+![Ricochet Programming overview](images/1-ricochet-overview.png)
+Assuming, that you are refining work pretty well, and create tasks for each user story. According to the timeline, developer 
+started the work from `Task A`. Once finished, `Task C` was started, and then `Task B`. During work on each of the tasks, 
+small pieces of code were added, that developer is unaware they might reflect other parts of code. Those ricochets, when
```

Comment:
maybe instead of 'small pieces of code' - 'smart **portions** of code'?

Reply:
Good clarification. Done